src/sympc/approximations/sigmoid.py

```diff
@@ -41,3 +44,30 @@ def sigmoid(tensor: MPCTensor, method: str = "exp") -> "MPCTensor":
             result += (tensor ** d) * weights[i + 1]
 
         return result
+
+    elif method == "chebyshev":
+        # Make sure the elements are all positive
+        _sign = sign(tensor)
+        tensor_8 = tensor * _sign / 8
+        p = 11
+        q = 11
+        scaler = ((1 + tensor_8) / 2) ** (q + 1)
+
+        def factorial(n):
+            fact = 1
+            for i in range(1, n + 1):
+                fact = fact * i
+            return fact
+
+        polynomial = 0 * tensor_8
+        for mu in range(p + 1):
+            a_n = factorial(mu + q) / (factorial(mu) * factorial(q))
+            T_n_w = ((1 - tensor_8) / 2) ** mu
+            polynomial += a_n * T_n_w * _sign
+
+        result = scaler * polynomial
+
+        return ((1 - _sign) * (1 + result) + (1 + _sign) * (result)) / 2
```

Comment:
so when it's negative we actually return `(1 + result)`, why is so ?

What is the impact if tensor_8 was not positive?

Reply:
### Please Click the Image Below
[<img src="https://user-images.githubusercontent.com/31875325/116693482-9c4a9e80-a9db-11eb-8039-49286ad43c70.png" width="500" height="500">](https://www.desmos.com/calculator/c5wcl6dzqy)

Reply:
I feel stupid - shouldn't it `(1 - result)` then, when sign is negative?

Reply:
I actually multiplied the sign earlier, hence leading to a double sign inversion.

https://github.com/OpenMined/SyMPC/pull/136/files#diff-4e30e76758ec501224f7729e0b211d42a81a73e390f68f83311c16f0c625bc75R67

I will remove both of them.

Reply:
ah great!!